pep-0514.txt

```diff
@@ -30,7 +30,9 @@ Motivation
 When installed on Windows, the official Python installer creates a registry key
 for discovery and detection by other applications. This allows tools such as
 installers or IDEs to automatically detect and display a user's Python
-installations.
+installations. For example, ``virtualenv`` supports a ``-p`` argument to use an
+alternative interpreter, but is unable to provide a list of all values that may
+be provided by a user.
```

Comment:
Just noticed that this hypothetical motivation isn't as motivating as the fact that py.exe and some IDEs are already trusting the information, despite the potential for collisions. So I'm changing this around slightly to emphasise that we're fixing a problem here, not adding a new feature.